lib/fog/vsphere/requests/compute/vm_clone.rb

```diff
@@ -165,7 +165,7 @@ def vm_clone(options = {})
             cust_ip_settings ||= RbVmomi::VIM::CustomizationIPSettings.new("ip" => RbVmomi::VIM::CustomizationDhcpIpGenerator.new())
             cust_ip_settings.dnsDomain = cust_domain
             cust_global_ip_settings = RbVmomi::VIM::CustomizationGlobalIPSettings.new
-            cust_global_ip_settings.dnsServerList = cust_ip_settings.dnsServerList
+	    cust_global_ip_settings.dnsServerList = cust_ip_settings.dnsServerList
```

Comment:
Small issue, but please don't replace spaces with tabs.  The code no longer lines up depending on tab-spacing.  If you can fix and rebase, that'd be helpful.